vue.config.js

```diff
@@ -37,6 +38,12 @@ module.exports = {
         title: 'cameraPermissions',
         filename: 'CameraRequestPermission.html',
       },
+      offscreen: {
+        template: 'src/popup/offscreen.html',
+        entry: 'src/popup/offscreen.ts',
```

Comment:
As already mentioned: I feel those files should be located in `src/offscreen` directory. What do you think?